### PR TITLE
fix getting points from PolyLineROI

### PR DESCRIPTION
**Pull request by vinsfan368**
The issue is `getState()` returns the _relative_ position of the `PolyLineROI`, for whatever reason—if the ROI is dragged after being made, the result of `getPoints()` will not change. We have to get the handle positions of the `PolyLineROI`, then map those positions to the parent image, since these coordinates are affected by zooming and panning. Sample image and trajs.csv file attached, if needed:
[mask_points_bug_MRE.zip](https://github.com/alecheckert/quot/files/13866222/mask_points_bug_MRE.zip)

### CODE REVIEW

**Review by alecheckert — APPROVED** [review]
Looks good and thanks for the sample data.

Remind me, do you have permission to merge this yourself? If so, go ahead once we've merged #10.

**Comment by vinsfan368** [comment]
Yup, I can merge! I wanted to get another pair of eyes on this, thanks a lot.

**Comment by alecheckert** [comment]
> Yup, I can merge! I wanted to get another pair of eyes on this, thanks a lot.

Absolutely, continue to do this through PRs + code review - just wanted to make sure you had the right permissions.

**Pull request merged**
